servers/curated-corpus-api/schema-public.graphql

```diff
@@ -139,6 +146,10 @@ type Query {
   This is a future improvement, not needed now.
   """
   surface(id: ID!): Surface!
+  """
+  Retrieves a list of active Sections with their corresponding active SectionItems for a scheduled surface.
+  """
+  getSectionsWithSectionItems(scheduledSurfaceGuid: ID!): [Section!]!
```

Comment:
thinking about how we implemented collections in the public graph...

for brevity and consistency, i think we should name this `getSections`. we don't qualify other query names with the child entities and it's probably overly verbose here.

and our convo (and your suggestion) when we met recently about filters for this query...i kind of think we should wrap `scheduledSurfaceGuid` in a `filters` param from the outset. it leaves us a nice opening for adding client customization without changing the query signature. what do you think?

Reply:
@jpetto yep I like the idea also wrapping this in a filter obj. should we do the same then for the admin graph?